src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -340,6 +340,16 @@ public void usage(final PrintStream stream, final boolean printCommon) {
         }
     }
 
+    private String convertFromHtml(final String textToConvert) {
```

Comment:
I noticed that there are some-times non-ascii characters embedded into the docs, could you look for these at this point and fail if this happens so that we do not have to rely on reviewers to catch this?